src/BillingBenefits/BillingBenefits.Autorest/README.md

```diff
@@ -55,10 +55,10 @@ subject-prefix: $(service-name)
 
 directive:
   - where:
-      variant: ^(Create|Update)(?!.*?(Expanded|JsonFilePath|JsonString))
+      variant: ^(Create|Update|Validate)(?!.*?(Expanded|JsonFilePath|JsonString))
     remove: true
   - where:
-      variant: ^CreateViaIdentity$|^CreateViaIdentityExpanded$|^ValidateViaIdentitySavingsPlanOrder$|^ValidateViaIdentitySavingsPlanOrderExpanded$
+      variant: ^CreateViaIdentity$|^CreateViaIdentityExpanded$|^ValidateViaIdentitySavingsPlanOrderExpanded$|^ElevateViaIdentity$
```

Comment:
Why to remove ^ValidateViaIdentitySavingsPlanOrderExpanded$|^ElevateViaIdentity$

Reply:
Removed the cmdlets “Invoke-AzBillingBenefitsElevateSavingPlanOrder”, “Invoke-AzBillingBenefitsSavingsPlanPurchaseValidation” and “Invoke-AzBillingBenefitsSavingsPlanUpdateValidation” which are mainly part of the syntax in “XXXViaIdentity”.
This is because these three cmdlets now generate syntax that supports “ValidateViaJsonFilePath”, “ValidateViaJsonString”.

Reply:
Removed the cmdlets “Invoke-AzBillingBenefitsElevateSavingPlanOrder”, “Invoke-AzBillingBenefitsSavingsPlanPurchaseValidation” and “Invoke-AzBillingBenefitsSavingsPlanUpdateValidation” which are mainly part of the syntax in “XXXViaIdentity”.
This is because these three cmdlets now generate syntax that supports “ValidateViaJsonFilePath”, “ValidateViaJsonString”.